docs/tutorials/google/start.ipynb

```diff
@@ -101,7 +101,7 @@
         "\n",
         "Access is currently restricted to those in an approved group, and you must be in that group before running this tutorial.\n",
         "\n",
-        "You can find more about running this in colaboratory in the [Colab documentation](https://colab.research.google.com/notebooks/welcome.ipynb) or in our Cirq-specific guide to [running in Colab](colab.ipynb).  You can download this notebook from the [GitHub repository](https://github.com/quantumlib/Cirq/blob/main/docs/tutorials/google/start.ipynb)."
+        "You can find more about running this in colaboratory in the [Colab documentation](https://colab.research.google.com/notebooks/welcome.ipynb) or in our Cirq-specific guide to [running in Colab](colab.ipynb).  You can download this notebook from the [GitHub repository](https://github.com/quantumlib/Cirq/blob/main/docs/tutorials/google/start.ipynb). It is recommended that your colab runs using Python 3.11 or greater."
```

Comment:
I think that this isn't an issue in Colab since they don't use gcloud. Abe ran into this in the [command line flow](https://quantumai.google/cirq/tutorials/google/start#running_from_the_command_line) while setting up auth, so maybe include it in the [auth section](https://quantumai.google/cirq/tutorials/google/start#setup_google_cloud_authentication)?